### PR TITLE
couple of minor fixes while building docs

### PR DESCRIPTION
**Pull request by interurban**

### CODE REVIEW

File: themes/default/content/docs/pulumi-cloud/deployments/drift.md [hunk 1]
```diff
@@ -66,22 +66,22 @@ You can integrate Drift notifications to Slack, MS Teams, and more using Pulumi
 
 ![Drift Webhook Events](../drift-webhook-events.png)
 
-Summary of Drift Events
+### Summary of Drift Detection events
```
Comment (cleverguy25) [inline themes/default/content/docs/pulumi-cloud/deployments/drift.md:69]: These are not just drift detection events, they are drift remediation as well, so this is actually wrong.

File: themes/default/content/docs/pulumi-cloud/deployments/drift.md [hunk 2]
```diff
@@ -12,35 +12,35 @@ menu:
...
 
-Any Preview of a Refresh is considered a drift detection run in Pulumi Cloud. Running drift detection from the CLI is as simple as running `pulumi refresh --preview-only` or even just `pulumi refresh`, which will first run a preview (thereby creating the drift run), before running the actual refresh.
+Any preview of a refresh is considered a drift detection run in Pulumi Cloud. Running Drift Detection from the CLI is as simple as running `pulumi refresh --preview-only` or even just `pulumi refresh`, which will first run a preview (thereby creating the drift run), before running the actual refresh.
 
 After your run completes you can see the drift run via the [Drift tab](#drift-tab) for your stack.
 
 ## Pulumi Cloud UI
 
-### Running via Click to Deploy
+### Running via click to deploy
```
Comment (cleverguy25) [inline themes/default/content/docs/pulumi-cloud/deployments/drift.md:27]: In other places Click to Deploy is capitalized, as it is a feature name. Why do we not want that here?

File: themes/default/content/docs/pulumi-cloud/deployments/drift.md [hunk 3]
```diff
@@ -12,35 +12,35 @@ menu:
...
 
 Regardless of how a drift run is initiated, all results will appear in this tab. If there is drift detected for a run, a diff summary of the changes will be included in the information card.
 
 ![Drift tab](../drift-tab.png)
 
-### Configuring Drift Detection on a Schedule
+### Configuring Drift Detection on a schedule
 
-In order to set up Drift Detection and Remediation in the Pulumi Cloud console, follow these steps:
+In order to set up Drift Detection and remediation in the Pulumi Cloud console, follow these steps:
```
Comment (cleverguy25) [inline themes/default/content/docs/pulumi-cloud/deployments/drift.md:43]: if we leave Drift Detection capitalized then Remediation should be capitalized, it we lower case remediation we should change to lower case drift detection.

File: themes/default/content/docs/pulumi-cloud/deployments/drift.md (hunk 1 above)
Comment (interurban) [reply themes/default/content/docs/pulumi-cloud/deployments/drift.md:69]: gotcha; i'll put it back. thx

File: themes/default/content/docs/pulumi-cloud/deployments/drift.md [hunk 4]
```diff
@@ -12,13 +12,13 @@ menu:
 
 Drift Detection is the process of identifying changes in the actual state of your cloud environment that deviate from the expected state stored in Pulumi Cloud. This deviation can occur for various reasons, including manual adjustments made directly in the cloud provider’s console, unintended consequences of scripts, or unauthorized changes.
 
-In order to detect and/or remediate drift via Pulumi Deployments, you are first required to configure deployment settings for your stack.
+To detect and remediate drift via Pulumi Deployments, you are first required to configure deployment settings for your stack.
```
Comment (meagancojocar) [inline themes/default/content/docs/pulumi-cloud/deployments/drift.md:15]:
```suggestion
To use Drift Detection and Remediation via Pulumi Deployments, you are first required to configure deployment settings for your stack.
```

**Review by meagancojocar — APPROVED** [review]

File: themes/default/content/docs/pulumi-cloud/deployments/drift.md [hunk 5]
```diff
@@ -12,13 +12,13 @@ menu:
...
+To use Drift Detection and Remediation via Pulumi Deployments, you are first required to configure deployment settings for your stack.
 
-You can also run a remediate drift operation, which will run a `pulumi up --refresh` to treat the pulumi program as the source of truth and override what is in the cloud.
+You can also run a remediate drift operation, which will run a `pulumi up --refresh` to treat the Pulumi program as the source of truth and override what is in the cloud.
 
-## Runing Drift detection from the CLI
+## Running Drift Detection from the CLI
 
-Any Preview of a Refresh is considered a drift detection run in Pulumi Cloud. Running drift detection from the CLI is as simple as running `pulumi refresh --preview-only` or even just `pulumi refresh`, which will first run a preview (thereby creating the drift run), before running the actual refresh.
+Any preview of a refresh is considered a drift detection run in Pulumi Cloud. Running Drift Detection from the CLI is as simple as running `pulumi refresh --preview-only` or even just `pulumi refresh`, which will first run a preview (thereby creating the drift run), before running the actual refresh.
```
Comment (aaronkao) [inline themes/default/content/docs/pulumi-cloud/deployments/drift.md:21]: Is drift detection a proper noun?

File: themes/default/content/docs/pulumi-cloud/deployments/drift.md [hunk 6]
```diff
@@ -66,22 +66,22 @@ You can integrate Drift notifications to Slack, MS Teams, and more using Pulumi
...
+### Setting it up via the REST API
 
 For those who prefer to automate and script their infrastructure tasks, Drift Detection and Remediation can be configured programmatically using simple HTTP requests. Here are the new endpoints we have added:
 
-* Create a Drift schedule
-* Get a Drift schedule
-* Update or delete a Drift schedule
-* Pause or resume a Drift schedule
+* Create a drift schedule
+* Get a drift schedule
```
Comment (aaronkao) [inline themes/default/content/docs/pulumi-cloud/deployments/drift.md:82]: Here you dont use it as a proper noun. May want to be more consistent

File: themes/default/content/docs/pulumi-cloud/deployments/drift.md [hunk 7]
```diff
@@ -12,13 +12,13 @@ menu:
 
 Drift Detection is the process of identifying changes in the actual state of your cloud environment that deviate from the expected state stored in Pulumi Cloud. This deviation can occur for various reasons, including manual adjustments made directly in the cloud provider’s console, unintended consequences of scripts, or unauthorized changes.
 
-In order to detect and/or remediate drift via Pulumi Deployments, you are first required to configure deployment settings for your stack.
+To use Drift Detection and Remediation via Pulumi Deployments, you are first required to configure deployment settings for your stack.
```
Comment (cnunciato) [inline themes/default/content/docs/pulumi-cloud/deployments/drift.md:15]:
```suggestion
To use drift detection and remediation with Pulumi Deployments, you must first configure the deployment settings for your stack.
```

File: themes/default/content/docs/pulumi-cloud/deployments/drift.md (hunk 5 above)
Comment (cnunciato) [reply themes/default/content/docs/pulumi-cloud/deployments/drift.md:21]: Products are proper nouns (Pulumi Deployments), features/capabilities should not be.

File: themes/default/content/docs/pulumi-cloud/deployments/drift.md [hunk 8]
```diff
@@ -12,13 +12,13 @@ menu:
...
 
-You can also run a remediate drift operation, which will run a `pulumi up --refresh` to treat the pulumi program as the source of truth and override what is in the cloud.
+You can also run a remediate drift operation, which will run a `pulumi up --refresh` to treat the Pulumi program as the source of truth and override what is in the cloud.
 
-## Runing Drift detection from the CLI
+## Running Drift Detection from the CLI
```
Comment (cnunciato) [inline themes/default/content/docs/pulumi-cloud/deployments/drift.md:19]:
```suggestion
## Running drift detection from the CLI
```

File: themes/default/content/docs/pulumi-cloud/deployments/drift.md (hunk 5 above)
Comment (cnunciato) [inline themes/default/content/docs/pulumi-cloud/deployments/drift.md:21]:
```suggestion
Any preview of a refresh is considered a drift detection run in Pulumi Cloud. Running drift detection from the CLI is as simple as running `pulumi refresh --preview-only` or even just `pulumi refresh`, which will first run a preview (thereby creating the drift run), before running the actual refresh.
```

File: themes/default/content/docs/pulumi-cloud/deployments/drift.md [hunk 9]
```diff
@@ -66,22 +66,22 @@ You can integrate Drift notifications to Slack, MS Teams, and more using Pulumi
 
 ![Drift Webhook Events](../drift-webhook-events.png)
 
-Summary of Drift Events
+### Summary of Drift Detection and Remediation events
```
Comment (cnunciato) [inline themes/default/content/docs/pulumi-cloud/deployments/drift.md:69]:
```suggestion
### Summary of drift detection and remediation events
```

**Review by cnunciato — COMMENTED** [review]
Someone from product should maybe chime in, but to me, these are features of the Pulumi Deployments product, not names of a separate product, so should not be capitalized.

**Comment by meagancojocar** [comment]
> Someone from product should maybe chime in, but to me, these are features of the Pulumi Deployments product, not names of a separate product, so should not be capitalized.

Let's capitalize for now.

**Pull request merged**